CNAME

```diff
@@ -0,0 +1 @@
+iree.dev
```

Comment:
Can you test how `mkdocs gh-deploy` interacts with this file? I'm wondering if that will overwrite the `gh-pages` branch and if we'll need to add this file to https://github.com/iree-org/iree/tree/main/docs/website/overrides instead (on the `main` branch).

https://github.com/iree-org/iree/tree/main/docs/website#how-to-edit-this-documentation

Reply:
https://www.mkdocs.org/user-guide/deploying-your-docs/#custom-domains that seems likely